check_sites.py

```diff
@@ -13,19 +13,35 @@
 # limitations under the License.
 from FpsCheck import FpsCheck
 import json
+import getopt
+import sys
 from publicsuffix2 import PublicSuffixList
 
 
 def main():
+    args = sys.argv[1:]
+    inputFile = 'first_party_sets.JSON'
+    inputPrefix = ''
+    opts, _ = getopt.getopt(args, "i:")
+    for opt, arg in opts:
+        if opt == '-i':
+            inputFile = arg
+            inputPrefix='main/'
+
     # Open the canonical sites, and load the json
-    with open('first_party_sets.JSON') as f:
-        fps_sites = json.load(f)
+    with open(inputFile) as f:
+        try:
+            fps_sites = json.load(f)
+        except Exception as inst:
+        # If the file cannot be loaded, we will not run any other checks
+            print(inst)
+            exit()      
 
     # Load the etlds from the public suffix list
-    etlds = PublicSuffixList(psl_file = 'effective_tld_names.dat')
+    etlds = PublicSuffixList(psl_file = inputPrefix+'effective_tld_names.dat')
```

Comment:
Please use `os.path.join`, since that doesn't assume that the path separator should be `/`. Same comment in the other places that use `inputPrefix`.

Reply:
Applied this suggestion